README.md

```diff
@@ -1,3 +1,5 @@
-# Northcoders News API
+###1 You will need to create two .env files for your project: .env.test and .env.development. 
+Into each, add PGDATABASE=, with the correct database name for that environment 
+(see /db/setup.sql for the database names).
```

Comment:
You could write those database names here to make it easier to set it up and try it.